statsmodels/tsa/ar_model.py

```diff
@@ -1,36 +1,36 @@
 # -*- coding: utf-8 -*-
-from statsmodels.compat.python import iteritems, lmap
+from statsmodels.compat.python import iteritems
```

Comment:
note to self: iteritems no longer needed

Reply:
removed this singleton